community/mission.md

```diff
@@ -0,0 +1,27 @@
+---
+breadcrumb: Mission and Values
+---
+# Our Mission and Values
+Our mission and values serve not just as statements on a page, but as a compass,
+directing our work, shaping our decisions, and reminding us of the purpose that
+brought us together. They embody our identity, our aspirations, and the ethical
+path we've chosen to navigate through the complex realms of privacy and security.
+As you read on, know that these are not mere words, but the living, breathing
+essence of who we are and what we steadfastly strive for.
```

Comment:
I don't think this additional text is necessary. I would keep it simple and just have this page contain the mission statement and values.

Reply:
from the one hand - my analytical mind doesn't need this additional "flowery" text, on the other - I think it is good to be explicit that we do apply m&v in ways described.

I wonder what @TamaraDale's mind would think about this. :-)

Reply:
Ultimately we use it for guiding our decisions. If we must say anything, just say that (as proposed in my other comment above)

Reply:
Yeah, I mostly agree with Matt that this is too much. OTOH just guiding our decisions could be interpreted that it influences only the OTC/OMC ruling body decisions so I would say that it is also a guidance of our daily work as members of the project.

Reply:
Removed this section entirely.